db/migrate/20171212101253_add_submit_external_to_message_thread.rb

```diff
@@ -0,0 +1,9 @@
+class AddSubmitExternalToMessageThread < ActiveRecord::Migration
+  def change
+    add_column :message_threads, :external_service_id, :integer
```

Comment:
If we use 
`add_reference :message_threads, :external_service, foreign_key: true`
it adds and index and preserves referential integrity (i.e. we can't have a thread referencing a non-existent external service.